tensorrt_llm/_torch/attention_backend/sparse/deepseek_v4/__init__.py

```
@@ -1,2 +1,4 @@
-# SPDX-FileCopyrightText: Copyright (c) 2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
+from .cache_manager import DeepseekV4CacheManager
+from .deepseek_v4 import DeepseekV4AttentionType
+
+__all__ = ["DeepseekV4CacheManager", "DeepseekV4AttentionType"]
```

Ruff (0.15.18)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)